Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -71,6 +71,12 @@ instance (priority := 100) isCyclic_of_subsingleton [Group α] [Subsingleton α]
 #align is_cyclic_of_subsingleton isCyclic_of_subsingleton
 #align is_add_cyclic_of_subsingleton isAddCyclic_of_subsingleton
 
+theorem isCyclic_multiplicative_iff [AddGroup α] : IsCyclic (Multiplicative α) ↔ IsAddCyclic α :=
```

Comment:
Can you add the `Additive` version too (and below)? Also, this seems sensible as a `simp` lemma.